oshc/main/templates/session.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'base.html' %} {% load static %} {% block content %}
 
 <iframe id="typeform-full" width="100%" height="100%" frameborder="0" src="https://oshc.typeform.com/to/CvcyEy"></iframe>
+
 <script type="text/javascript" src="https://embed.typeform.com/embed.js"></script>
 
 {% endblock %}
-<html>
```

Comment:
Extraneous change. Please do not modify the files which are not related to the issue.